.github/workflows/build-test-ubuntu-x64.yml

```diff
@@ -170,7 +170,9 @@ jobs:
           LD_LIBRARY_PATH: .
           USE_MESHLIB2_PY: 1
         working-directory: build/${{ matrix.config }}/bin
-        run: python3 ./../../../scripts/run_python_test_script.py -d '../test_python'
+        run: \
+          python3 ./../../../scripts/run_python_test_script.py -d '../test_python'
+          python3 ./../../../scripts/run_python_test_script.py -d '../test_regression'
```

Comment:
Regression tests starts in further steps, after coping test data from S3
It will fail here.